js/packages/teams-ai/src/DefaultTurnStateManager.ts

```diff
@@ -36,6 +36,13 @@ export interface DefaultUserState {}
  * state.
  */
 export interface DefaultTempState {
+    /**
+     * Token returned if the Application was configured with authentication support.
+     */
+    authTokens: {
```

Comment:
Clarifying question: Does this mean that users will be able to do multiple auth events that need the token in one turn? And then, that means every new turn will need a new auth request/token, correct?

Reply:
For signing in, yes. Once the user is signed in the bot will simply get the 2+ tokens from the token store with 2 method calls. This will happen in within the same turn.